config/src/main/java/org/springframework/security/config/authentication/PasswordEncoderParser.java

```diff
@@ -69,12 +69,12 @@ private void parse(Element element, ParserContext parserContext) {
 			this.passwordEncoder = new RuntimeBeanReference(ref);
 		}
 		else {
-			this.passwordEncoder = createPasswordEncoderBeanDefinition(hash, useBase64);
```

Comment:
It appears that `ATT_BASE_64` is no longer necessary. Will you please remove that too while you are at it?

Reply:
Hey @jzheaux . Sorry for the late comment, I didn't get the notification of your message. Will do it today.

Reply:
Done!